src/test/java/org/jenkinsci/plugins/github_branch_source/GithubAppCredentialsTest.java

```diff
@@ -455,41 +463,45 @@ public void testAgentRefresh() throws Exception {
           credentialsLog,
           contains(
               // (agent log added out of order, see below)
-              "Generating App Installation Token for app ID 54321 on agent", // 1
-              "Failed to generate new GitHub App Installation Token for app ID 54321 on agent: cached token is stale but has not expired", // 2
-              "Generating App Installation Token for app ID 54321 on agent", // 3
+              "Generating App Installation Token for app ID 54321", // 1
+              "Generating App Installation Token for app ID 54321", // 2
+              "Failed to generate new GitHub App Installation Token for app ID 54321: cached token is stale but has not expired", // 3
               // node ('my-agent') {
               // checkout scm
-              "Generating App Installation Token for app ID 54321",
-              // checkout scm
-              // (No token generation)
-              // sleep
               // checkout scm
               "Generating App Installation Token for app ID 54321",
               // (error forced by wiremock)
               "Failed to generate new GitHub App Installation Token for app ID 54321: cached token is stale but has not expired",
               // (error forced by wiremock - failed refresh on the agent)
               // "Generating App Installation Token for app ID 54321 on agent", // 1
-              "Generating App Installation Token for app ID 54321 for agent",
```

Comment:
Ahh..

@Olamy this is incorrect - the test was failing due to JENKINS-69488 which is why my PR is breaking this test :)

Reply:
good catch